src/Contrib.KafkaFlow.ProcessManagers.SqlServer/schema/0002.Table.sql

```diff
@@ -0,0 +1,18 @@
+/* Table */
+IF OBJECT_ID(N'[kafkaflowtest].[dbo].[processes]', N'U') IS NULL
+BEGIN
+    CREATE TABLE [kafkaflowtest].[dbo].[processes](
```

Comment:
Same comment as above with the `outbox` table regarding the database name and schema name.